docs/_data/table/en/common-configuration.yml

```diff
@@ -344,3 +356,113 @@ fs.swift.tenant:
   'Swift user for authentication.'
 fs.swift.user:
   'Swift tenant for authentication.'
+ftp.nonProxyHosts:
+  'N/A'
+gopherProxySet:
+  'N/A'
+http.nonProxyHosts:
+  'N/A'
+java.awt.graphicsenv:
+  'N/A'
+java.awt.printerjob:
+  'N/A'
+java.class.path:
+  'N/A'
+java.class.version:
+  'N/A'
+java.endorsed.dirs:
+  'N/A'
+java.ext.dirs:
+  'N/A'
+java.home:
+  'N/A'
+java.io.tmpdir:
+  'N/A'
+java.library.path:
+  'N/A'
+java.net.preferIPv4Stack:
+  'N/A'
+java.runtime.name:
+  'N/A'
+java.runtime.version:
+  'N/A'
+java.specification.name:
+  'N/A'
+java.specification.vendor:
+  'N/A'
+java.specification.version:
+  'N/A'
+java.vendor:
+  'N/A'
+java.vendor.url:
+  'N/A'
+java.vendor.url.bug:
+  'N/A'
+java.version:
+  'N/A'
+java.vm.info:
+  'N/A'
+java.vm.name:
+  'N/A'
+java.vm.specification.name:
+  'N/A'
+java.vm.specification.vendor:
+  'N/A'
+java.vm.specification.version:
+  'N/A'
+java.vm.vendor:
+  'N/A'
+java.vm.version:
+  'N/A'
+line.separator:
+  'N/A'
+log4j.configuration:
+  'N/A'
+org.apache.jasper.compiler.disablejsr199:
+  'N/A'
+org.apache.ratis.thirdparty.io.netty.allocator.useCacheForAllThreads:
+  'N/A'
+os.arch:
+  'N/A'
+os.name:
+  'N/A'
+os.version:
+  'N/A'
+path.separator:
+  'N/A'
+socksNonProxyHosts:
+  'N/A'
+sun.arch.data.model:
+  'N/A'
+sun.boot.class.path:
+  'N/A'
+sun.boot.library.path:
+  'N/A'
+sun.cpu.endian:
+  'N/A'
+sun.cpu.isalist:
+  'N/A'
+sun.io.unicode.encoding:
+  'N/A'
+sun.java.command:
+  'N/A'
+sun.java.launcher:
+  'N/A'
+sun.jnu.encoding:
+  'N/A'
+sun.management.compiler:
+  'N/A'
+sun.os.patch.level:
+  'N/A'
+user.country:
+  'N/A'
+user.dir:
+  'N/A'
+user.home:
+  'N/A'
+user.language:
+  'N/A'
+user.name:
+  'N/A'
+user.timezone:
+  'N/A'
```

Comment:
Do you know what added all these?

Reply:
we need to investigate this. it is also generated on my side for unknown reasons. 
I removed them manually for now